app/views/submission_mailer/confirmation.html.haml

```diff
@@ -0,0 +1,11 @@
+%h3 
+"Szanowna Pani / Szanowny Panie #{@submission.corresponding_author.surname}"
```

Comment:
To nie zadziała, bo correspoding_authro to niestety `string` a nie obiekt typu `Person`.